ktor-server/ktor-server-plugins/ktor-server-websockets/jvmAndNix/test/io/ktor/tests/websocket/CIOWebSocketTest.kt

```diff
@@ -4,11 +4,52 @@
 
 package io.ktor.tests.websocket
 
+import io.ktor.serialization.kotlinx.*
+import io.ktor.server.application.*
 import io.ktor.server.cio.*
+import io.ktor.server.routing.*
+import io.ktor.server.websocket.*
+import io.ktor.util.*
+import io.ktor.utils.io.*
+import io.ktor.websocket.*
+import kotlinx.serialization.Serializer
+import kotlinx.serialization.json.*
+import kotlin.test.*
 
+@InternalAPI
```

Comment:
why this annotation is needed here?

Reply:
Because I use `writeFrame(Frame.Close())`, which I copied from other tests.